opendebates/settings.py

```diff
@@ -260,3 +261,7 @@ def __contains__(self, item):
 DBBACKUP_STORAGE_OPTIONS = {'location': '.'}
 DBBACKUP_FILENAME_TEMPLATE = 'local/{datetime}.{extension}'
 DBBACKUP_SEND_EMAIL = False
+
+# Dummy settings for test etc
+NORECAPTCHA_SITE_KEY = ''
+NORECAPTCHA_SECRET_KEY = ''
```

Comment:
The norecaptcha library provides a shortcut too, which you can see in use in some of the tests.